### PR TITLE
Add select2@4.0.0

### PR DESCRIPTION
**Pull request by nicolashemonic**

### CODE REVIEW

File: package-overrides/github/cruikshj/knockout-select2@1.0.0.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "main": "dist/knockout-select2",
     "dependencies": {
-        "jquery": "npm:jquery@^2.2.0",
-        "knockout": "github:knockout/knockout@^3.3.0"
```
Comment (guybedford) [inline package-overrides/github/cruikshj/knockout-select2@1.0.0.json]: Do we really need to relax these versions?

Comment (nicolashemonic) [inline package-overrides/github/cruikshj/knockout-select2@1.0.0.json]: It does not specifically depends on these versions, so it's better not to specify ?

Comment (guybedford) [inline package-overrides/github/cruikshj/knockout-select2@1.0.0.json]: Will it work with jQuery 3 and knockout 4?

Comment (nicolashemonic) [inline package-overrides/github/cruikshj/knockout-select2@1.0.0.json]: I did not know most of package override should specify versions... Ok!

**Comment by sberney** [comment]
Your registry update broke my build. Does it really need jquery 2.2.0, or will any jquery 2 suffice?

**Comment by sberney** [comment]
The breakage had nothing to do with which jquery 2 will suffice. We kept getting the errors:
```
     Downloading npm:jquery@2.2.1
warn Error on getPackageConfig for npm:jquery
     Package.json meta not provided in endpoint request
warn Error on download for npm:jquery
     TypeError: Cannot read property 'dist' of undefined
         at NPMLocation.download (D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\jspm-npm\lib\npm.js:338:40)
         at then.then.active (D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\jspm\lib\registry.js:117:30)
         at lib$rsvp$$internal$$tryCatch (D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\rsvp\dist\rsvp.js:1036:16)
         at lib$rsvp$$internal$$invokeCallback (D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\rsvp\dist\rsvp.js:1048:17)
         at D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\rsvp\dist\rsvp.js:331:11
         at Object.lib$rsvp$asap$$flush [as _onImmediate] (D:\workspace\mira-dist\mira-dist-2016-02-25-test-build\App\Content\node_modules\rsvp\dist\rsvp.js:1198:9)
         at processImmediate [as _immediateCallback] (timers.js:345:15)
warn Error getting package config for npm:jquery.
warn Error downloading npm:jquery.
```

**Comment by guybedford** [comment]
@sberney which version of jspm and jspm npm is this with?

**Comment by sberney** [comment]
jspm-npm 0.26.3
jspm 0.16.25

**Comment by guybedford** [comment]
@sberney this sounds like a corrupted output from npm... Perhaps try clearing your cache via `jspm cc`? Are you running a local npm instance by any chance or is this the main registry?

**Pull request closed without merging**